core/application/app_configuration.hpp

```diff
@@ -162,6 +163,8 @@ namespace kagome::application {
     virtual OffchainWorkerMode offchainWorkerMode() const = 0;
 
     virtual bool isOffchainIndexingEnabled() const = 0;
+
+    virtual std::optional<primitives::BlockId> recoveryState() const = 0;
```

Comment:
```suggestion
    virtual std::optional<primitives::BlockId> recoverState() const = 0;
```

Reply:
Please feel the difference between recovery and recover. Pick the correct one here and further